src/extensions/styles/migrators/inlineDCLinkMigrator.js

```
@@ -38,29 +38,28 @@ const maxiVersions = [
 
 const migrate = attributes => {
-	const newAttributes = { ...attributes };
-
-	// Instead of saving saving the inline link status, save selected field as link target and enable the link
-	delete newAttributes['dc-post-taxonomy-links-status'];
-	newAttributes['dc-link-target'] = newAttributes['dc-field'];
-	newAttributes['dc-link-status'] = true;
+	// Direct property mutations for better performance
+	attributes['dc-link-target'] = attributes['dc-field'];
+	attributes['dc-link-status'] = true;
+	delete attributes['dc-post-taxonomy-links-status'];
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)